src/Symfony/Component/ErrorHandler/Resources/views/exception.html.php

```diff
@@ -16,7 +16,7 @@
 
     <div class="exception-message-wrapper">
         <div class="container">
-            <h1 class="break-long-words exception-message<?= mb_strlen($exceptionMessage) > 180 ? ' long' : ''; ?>"><?= $this->formatFileFromText(nl2br($exceptionMessage)); ?></h1>
+            <h1 class="break-long-words exception-message<?= mb_strlen((string) $exceptionMessage) > 180 ? ' long' : ''; ?>"><?= $this->formatFileFromText(nl2br($exceptionMessage)); ?></h1>
```

Comment:
can be removed also, the exception message cannot be null

Reply:
done